solr/core/src/java/org/apache/solr/handler/IndexFetcher.java

```diff
@@ -1066,7 +1066,7 @@ private void openNewSearcherAndUpdateCommitPoint() throws IOException {
 
   private void reloadCore() {
     final CountDownLatch latch = new CountDownLatch(1);
-    new Thread(
+    Thread.ofVirtual().name("CoreReload").start(
```

Comment:
Same here. Shouldn't this also use `unstarted()` per the table?

Reply:
See my other reply